### ISSUE TITLE
Can only run the Python Module in sudo

### ISSUE THREAD

**Issue by lessiar** [rmjarvis/TreeCorr#48]
Hello, I downloaded the code and installed it without errors in sudo (and also, not in sudo), and when I tried importing it to Python, I got this error:

```
>>> import treecorr
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/anaconda2/lib/python2.7/site-packages/treecorr/__init__.py", line 30, in <module>
    _lib = _ffi.dlopen(lib_file)
  File "/home/user/anaconda2/lib/python2.7/site-packages/cffi/api.py", line 139, in dlopen
    lib, function_cache = _make_ffi_library(self, name, flags)
  File "/home/user/anaconda2/lib/python2.7/site-packages/cffi/api.py", line 769, in _make_ffi_library
    backendlib = _load_backend_lib(backend, libname, flags)
  File "/home/user/anaconda2/lib/python2.7/site-packages/cffi/api.py", line 758, in _load_backend_lib
    return backend.load_library(name, flags)
OSError: cannot load library /home/user/anaconda2/lib/python2.7/site-packages/treecorr/_treecorr.so: /home/user/anaconda2/lib/python2.7/site-packages/treecorr/_treecorr.so: undefined symbol: GOMP_parallel

```
However, this error doesn't show up when using sudo python, so I don't know what's going on here.

**Comment by rmjarvis** [comment]
It looks like the problem is something with the gomp library.  Maybe that library on your system is not readable by you for some reason, but with sudo it is readable?  Just guessing here.  You'd need to use ldd on the _treecorr.so file to try to figure out what libgomp.so file it's trying to load and then see why it can't find it.

Here is what I get on one of my systems:
```
$ ldd ~/lib/python2.7/site-packages/TreeCorr-3.3.3-py2.7-linux-x86_64.egg/treecorr/_treecorr.so 
	linux-vdso.so.1 =>  (0x00007ffc48c8c000)
	libpython2.7.so.1.0 => /opt/astro/SL64/lib/libpython2.7.so.1.0 (0x00007fc8f723c000)
	libstdc++.so.6 => /usr/lib64/libstdc++.so.6 (0x00007fc8f6f11000)
	libm.so.6 => /lib64/libm.so.6 (0x00007fc8f6c8d000)
	libgomp.so.1 => /usr/lib64/libgomp.so.1 (0x00007fc8f6a7f000)
	libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007fc8f6869000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007fc8f664c000)
	libc.so.6 => /lib64/libc.so.6 (0x00007fc8f62b7000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00007fc8f60b3000)
	libutil.so.1 => /lib64/libutil.so.1 (0x00007fc8f5eb0000)
	/lib64/ld-linux-x86-64.so.2 (0x00007fc8f78ca000)
	librt.so.1 => /lib64/librt.so.1 (0x00007fc8f5ca7000)
```
So see what the corresponding command gives for you and do `ls -l` on the file pointed to for libgomp.so and make sure it is readable by you.

To be honest, I don't use sudo for installing it on linux ever, so I don't have much experience with how that can mess it up.  

Did you install it with pip, or from source?  In either case, you should be able to install without sudo.  The commands I typically use are:

    pip install treecorr --user --upgrade
    python setup.py install --prefix=~

To test these options, you might need to `sudo rm` the ones you installed in the anaconda directory first, since they'll take precedence.

**Comment by lessiar** [comment]
I initially installed it with pip (and sudo pip), but I think that created some conflicts with anaconda, since I only have it installed locally and I suppose that generates some clashing libraries. To fix that, I installed it with setup.py .
To do that I used ` python setup.py install --prefix=/home/user/anaconda2/lib ` to specify that I'd be installing it inside anaconda. I also added:
```
PYTHONPATH="${PYTHONPATH}:/home/user/anaconda2/lib/python2.7/site-packages/"
export PYTHONPATH 

```
 to .bashrc, since It appears I didn't have my PYTHONPATH specified.
It seems to be working now, thanks!
